docker/devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# The devcontainer Dockerfile to set up an hermetic enviornment with all needed dependencies.
+
+FROM rayproject/ray-deps:2023-06-26-cu116-aarch64
+COPY . /ray
+# Install dependencies needed to build ray
+RUN sudo apt-get update && sudo apt-get install -y curl unzip cmake gcc g++ && sudo apt-get clean
+RUN sudo chown -R ray:users /ray && cd /ray && git init && ./ci/env/install-bazel.sh --system
```

Comment:
local dev should not be using the `ci` scripts. I think what you need is just to install bazelisk.